app/services/ipam/base.rb

```diff
@@ -17,13 +17,14 @@ def initialize(opts = {})
       @mac = nil if @mac.try(:blank?)
       @errors = ActiveModel::Errors.new(self)
       @block_ip_minutes = opts.fetch(:block_ip_minutes, BLOCK_IP_MINUTES_DEFAULT)
+      @ip_addr = IPAddr.new("#{subnet.network}/#{subnet.mask}")
```

Comment:
In the other call we also include `subnet.family` to force the right family.
```suggestion
      @ip_addr = IPAddr.new("#{subnet.network}/#{subnet.mask}", subnet.family)
```

Reply:
Done